### ISSUE TITLE
Graph Designer: change alt text for add node when editing a Resource Model's graph

### ISSUE THREAD

**Issue by mradamcox** [archesproject/arches#1036]
Currently, this is what pops up in the graph designer:
![image](https://cloud.githubusercontent.com/assets/10427268/19608516/8eaa09a6-9797-11e6-9724-4aca835f8bdb.png)

(if picture upload doesn't work, I'm talking about the "Add Child Nodes" alt text)

I think it would be more clear this text were changed to "Add Branch" or something to that effect, because when users are editing the graph of a Resource Model conceptually it makes more sense to think of adding Branches--even though they technically are "adding child nodes".

**Comment by apeters** [comment]
I agree that this wording should be changed.  Do you agree @dwuthrich ?

**Comment by dwuthrich** [comment]
@apeters Yes, I agree

**Comment by apeters** [comment]
fixed by #2675
